src/signalrclient/logger.h

```diff
@@ -15,12 +15,27 @@ namespace signalr
     public:
         logger(const std::shared_ptr<log_writer>& log_writer, trace_level trace_level) noexcept;
 
+        void log(trace_level level, const char* entry, size_t entry_count) const;
+
+        // TODO: variadic 'const char*' overload to avoid std::string allocations
```

Comment:
Should we remove the todo and create an issue? Or just remove the todo?

Reply:
🤷‍♀️ I've been using `// TODO:` comments throughout the code base for all the small things that can be fixed up. I feel like there would be too many small issues if we instead made issues.

I'm planning on going through all the TODO's and doing something about them later in the development cycle, also tracking that with https://github.com/dotnet/aspnetcore/issues/8720#issue-423948619

>Resolve "TODO" comments